app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,11 @@
+<% provide(:title, "About") %>
```

Comment:
1. bỏ ngoặc `()` của `provide`
2. Sử dụng I18n cho toàn bộ file html
3. Nhớ dùng tag của rails
ví dụ: `a` thì dùng `link_tag`